Source/WebCore/animation/WebAnimationUtilities.cpp

```diff
@@ -32,8 +32,10 @@
 #include "AnimationPlaybackEvent.h"
 #include "CSSAnimation.h"
 #include "CSSAnimationEvent.h"
+#include "CSSParserContext.h"
 #include "CSSPropertyNames.h"
 #include "CSSSelector.h"
+#include "CSSSelectorParser.h"
```

Comment:
```suggestion
#include "CSSSelectorParserContext.h"
```
I fixed this in a follow up, but this should probably be addressed here